src/screens/Home/Home.js

```diff
@@ -630,6 +691,20 @@ class HomeScreen extends React.Component<Props, State> {
 
             <RecentConnectionsSpacer />
           }
+          {badges && badges.length ?
+            <BadgesWrapper>
+              <BadgesBlock>
+                <View style={{ backgroundColor: baseColors.snowWhite }}>
+                  <BadgesSubtitle subtitle title="game of badges." />
+                </View>
+                <BadgesScrollView horizontal nestedScrollEnabled overScrollMode="always">
+                  {this.renderBadges()}
+                </BadgesScrollView>
+              </BadgesBlock>
+            </BadgesWrapper> :
+
+            <BadgesSpacer />
```

Comment:
Or you could not render here anything and set stickyHeaderIndices to 3 only if badges exists